### DISCUSSION TITLE
Downloading spacy models

### DISCUSSION THREAD

**Discussion by jh-139** [explosion/spaCy#12500] [Help: Installation] [answered]
Hi i am trying to download spacy model on a linux server but however the server has blocked access to the internet. However
i may request to the network administrator to grant access to certain websites.

I am trying to download the spacy model with this command: python -m spacy download en_core_web_trf .  What websites do i need to request access in order for this command to work?

Thanks!

**Comment by shadeMe** [comment, marked as answer]
spaCy models are hosted on [GitHub](https://github.com/explosion/spacy-models), so you'll need to ask your administrator to white-list it. Here's an example URL:
```
https://github.com/explosion/spacy-models/releases/download/de_dep_news_trf-3.5.0/de_dep_news_trf-3.5.0-py3-none-any.whl
```
